src/Options/Features.php

```diff
@@ -75,23 +76,44 @@ public function get_features( string $feature = '' ): array {
 	 * @return boolean True if the feature is enabled, false if disabled.
 	 */
 	public function is_enabled( string $feature ): bool {
-		$option = $this->get_features();
+		$config = $this->get_features( $feature );
 
-		if ( isset( $option['features'][ $feature ]['enabled'] ) ) {
-			return $option['features'][ $feature ]['enabled'];
+		if ( isset( $config['enabled'] ) ) {
+			return $config['enabled'];
 		}
 
 		return false;
 	}
 
+	/**
+	 * Return the default value for a feature.
+	 *
+	 * @param string $feature
+	 * @return mixed
+	 */
+	public function default_value( string $feature ) {
+		$config = $this->get_features( $feature );
+
+		if ( isset( $config['attributes']['default'] ) ) {
+			return $config['attributes']['default'];
+		}
+
+		return null;
```

Comment:
```suggestion
return $config['attributes']['default'] ?? null;
```

Null coalescing operator can be used here.

Reply:
This has been updated